in_text/text_04_predictions.yml

```diff
@@ -76,800 +76,1124 @@ cross-cites:
+      data-units: days since 
     -
-      attr-label: TMP
+      attr-label: time
       attr-def: >-
-        Air temperature
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: degrees C
+        Forecast date
+      attr-defs: This data release
+      data-min: XX # will complete when we have latest vbersion of this dataset
+      data-max: XX # will complete when we have latest vbersion of this dataset
+      data-units: degrees Celsius
     -
-      attr-label: DSWRF
-      attr-def: >-
-        Downward short-wave radiation flux
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
+      attr-label: model_name
+      attr-def:  >-
+        Model name. DA, no DA and persistence
+      attr-defs: This data release
       data-min: NA
       data-max: NA
-      data-units: Watts per meter squared
+      data-units: NA
     -
-      attr-label: WS
-      attr-def: >-
-        Wind speed
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: meters per second
+      attr-label: max_temp_c_predicted
+      attr-def:  >-
+        Maximum temperature predicted in degrees C
+      attr-defs: This data release
+      data-min: 6.4 # will update when we have latest vbersion of this dataset
+      data-max: 24.6 # will update when we have latest vbersion of this dataset
+      data-units: degree C
     -
-      attr-label: TMAX
-      attr-def: >-
-        Maximum air temperature
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: degrees C
+      attr-label: max_temp_l90
+      attr-def:  >-
+        XX
+      attr-defs: This data release 
+      data-min: 5.86 # will update when we have latest vbersion of this dataset
+      data-max: 22.49 # will update when we have latest vbersion of this dataset
+      data-units: degree C
     -
-      attr-label: TMIN
-      attr-def: >-
-        Minimum air temperature
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: degrees C
+      attr-label: max_temp_u90
+      attr-def:  >-
+        XX
+      attr-defs: This data release
+      data-min: 8.03 # will update when we have latest vbersion of this dataset
+      data-max: 25.03 # will update when we have latest vbersion of this dataset
+      data-units: degree C
     -
-      attr-label: APCP
-      attr-def: >-
-        Total accumulated precipitation
-      attr-defs: https://www.nco.ncep.noaa.gov/pmb/products/gens/gec00.t00z.pgrb2s.0p25.f003.shtml
-      data-min: NA
-      data-max: NA
-      data-units: meters
+      attr-label: prob_exceed_75
+      attr-def:  >-
+        XX
```

Comment:
Not sure how to describe this here, @jzwart @limnoliver @jdiaz4302 do you know how we should describe this?

Reply:
This is the predicted probability (in decimal fraction) of stream temperature exceeding 75 degrees F for a given segment and date. It's calculated as the fraction of ensemble predictions that exceeded 75 F.